src/rollup/types.d.ts

```diff
@@ -265,10 +322,10 @@ export interface RollupCache {
 	modules: ModuleJSON[];
 }
 
-export interface Bundle {
+export interface RollupFileBuild {
```

Comment:
This name confused me at first as we are always building files. Maybe `RollupSingleFileBuild` as the "single" part ?